Specify oracledb version for blackbox tests

licitdev opened the pull request:
A very recent release of `node-oracledb` to version `6.0.0` from `5.5.0` had caused issues with our OracleDB blackbox tests.
Ref: https://node-oracledb.readthedocs.io/en/latest/release_notes.html#node-oracledb-v6-0-0-24-may-2023

licitdev committed:
Specify oracledb version for blackbox tests

paescuj reviewed (approved):
Makes sense!

(Do you know which change is responsible for the issues?)

licitdev commented:
> Do you know which change is responsible for the issues?

@paescuj No clue at the moment~ 🙈  

<details>
  <summary>Attaching error logs for future reference</summary>
  
  ```
  Run TEST_DB=oracle pnpm run test:blackbox

  > directus-monorepo@ test:blackbox /home/runner/work/directus/directus
  > pnpm --filter directus deploy --prod dist && pnpm --filter tests-blackbox test

  Packages are copied from the content-addressable store to the virtual store.
    Content-addressable store is at: /home/runner/setup-pnpm/node_modules/.bin/store/v3
    Virtual store is at:             dist/node_modules/.pnpm
  Progress: resolved 1, reused 0, downloaded 0, added 0
  api                                      |  WARN  deprecated keyv-memcache@1.3.3
  api                                      |  WARN  deprecated json2csv@5.0.[7](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:8)
  Progress: resolved 153, reused 123, downloaded 21, added 0
  app                                      |  WARN  deprecated @joeattardi/emoji-button@4.6.4
  api                                      |  WARN  deprecated @types/keyv@4.2.0
  Progress: resolved 313, reused 291, downloaded 21, added 0
  Progress: resolved 402, reused 376, downloaded 21, added 0
  Progress: resolved 636, reused 420, downloaded 21, added 0
  api                                      |  WARN  deprecated @types/cssnano@5.1.0
  Progress: resolved [8](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:9)04, reused 531, downloaded 21, added 0
  Progress: resolved [9](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:10)77, reused 559, downloaded 21, added 0
  directus                                 |  WARN  deprecated request@2.88.2
  Progress: resolved [10](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:11)90, reused 652, downloaded 21, added 0
  Progress: resolved [12](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:13)03, reused 675, downloaded 21, added 0
  api                                      |  WARN  deprecated uuid@3.4.0
  directus                                 |  WARN  deprecated har-validator@5.1.5
  directus                                 |  WARN  deprecated @npmcli/move-file@1.1.2
  Progress: resolved 1433, reused 778, downloaded 21, added 0
  api                                      |  WARN  deprecated stable@0.1.8
  Progress: resolved 1665, reused 960, downloaded 21, added 0
  Progress: resolved 1830, reused 1477, downloaded 21, added 0
  Progress: resolved 1937, reused 1834, downloaded 21, added 0
  Progress: resolved 2178, reused 2099, downloaded 21, added 0
  packages/specs                           |  WARN  deprecated superagent@7.1.6
  Progress: resolved 2460, reused 2405, downloaded 21, added 0
  Progress: resolved 2468, reused 24[13](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:14), downloaded 21, added 0
  .                                        |    +1094 ++++++++++++++++++++++++++++
  Progress: resolved 2468, reused 24[14](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:15), downloaded 21, added 130
  Progress: resolved 2468, reused 2414, downloaded 21, added 552
  Progress: resolved 2468, reused 2414, downloaded 21, added 1094, done
   WARN  Failed to create bin at /home/runner/work/directus/directus/dist/node_modules/.pnpm/vite@4.3.7_@types+node@20.2.0_sass@1.62.1/node_modules/vite/node_modules/.bin/sass. The source file at /home/runner/work/directus/directus/dist/node_modules/.pnpm/sass@1.62.1/node_modules/sass/sass.js does not exist.
   WARN  Issues with peer dependencies found
  api
  └─┬ @directus/extensions-sdk 10.1.1
    └─┬ rollup-plugin-styles 4.0.0
      └── ✕ unmet peer rollup@^2.63.0: found 3.22.0 in @directus/extensions-sdk

  app
  └─┬ @directus/extensions-sdk 10.1.1
    └─┬ rollup-plugin-styles 4.0.0
      └── ✕ unmet peer rollup@^2.63.0: found 3.22.0 in @directus/extensions-sdk

  directus
  └─┬ @directus/api 10.2.0
    └─┬ @directus/extensions-sdk 10.1.1
      └─┬ rollup-plugin-styles 4.0.0
        └── ✕ unmet peer rollup@^2.63.0: found 3.22.0 in @directus/extensions-sdk

  packages/create-directus-extension
  └─┬ @directus/extensions-sdk 10.1.1
    └─┬ rollup-plugin-styles 4.0.0
      └── ✕ unmet peer rollup@^2.63.0: found 3.22.0 in @directus/extensions-sdk

  packages/extensions-sdk
  └─┬ rollup-plugin-styles 4.0.0
    └── ✕ unmet peer rollup@^2.63.0: found 3.22.0


  > tests-blackbox@ test /home/runner/work/directus/directus/tests/blackbox
  > jest




  👮‍♀️ Starting tests!

  ❯ Bootstrap databases and start servers
  ❯ OracleDB
  ✖ OracleDB [FAILED: Directus-oracle bootstrap failed: 
  Error: select * from "directus_fields" where "interface" = :1 - ORA-01002: fetch out of sequence
      at Protocol._processMessage (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/thin/protocol/protocol.js:167:17)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async ThinConnectionImpl.execute (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/thin/connection.js:799:7)
      at async Connection.execute (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/connection.js:858:16)
      at async Connection.<anonymous> (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/util.js:160:14) {
    offset: 0,
    errorNum: 1002,
    code: 'ORA-01002'
  }
  ]
  ✖ Bootstrap databases and start servers [FAILED: Directus-oracle bootstrap failed: 
  Error: select * from "directus_fields" where "interface" = :1 - ORA-01002: fetch out of sequence
      at Protocol._processMessage (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/thin/protocol/protocol.js:167:17)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async ThinConnectionImpl.execute (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/thin/connection.js:799:7)
      at async Connection.execute (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/connection.js:858:16)
      at async Connection.<anonymous> (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/util.js:160:14) {
    offset: 0,
    errorNum: 1002,
    code: 'ORA-01002'
  }
  ]
  Error: Jest: Got error running globalSetup - /home/runner/work/directus/directus/tests/blackbox/setup/setup.ts, reason: Error: Directus-oracle bootstrap failed: 
  Error: select * from "directus_fields" where "interface" = :1 - ORA-01002: fetch out of sequence
      at Protocol._processMessage (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/thin/protocol/protocol.js:167:17)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async ThinConnectionImpl.execute (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/thin/connection.js:799:7)
      at async Connection.execute (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/connection.js:858:16)
      at async Connection.<anonymous> (/home/runner/work/directus/directus/node_modules/.pnpm/oracledb@6.0.0/node_modules/oracledb/lib/util.js:160:14) {
    offset: 0,
    errorNum: 1002,
    code: 'ORA-01002'
  }

      at /home/runner/work/directus/directus/tests/blackbox/setup/setup.ts:174:[15](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:16)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async exports.default (/home/runner/work/directus/directus/tests/blackbox/setup/setup.ts:49:5)
      at async /home/runner/work/directus/directus/node_modules/.pnpm/@jest+core@29.5.0/node_modules/@jest/core/build/runGlobalHook.js:109:13
      at async waitForPromiseWithCleanup (/home/runner/work/directus/directus/node_modules/.pnpm/@jest+transform@29.5.0/node_modules/@jest/transform/build/ScriptTransformer.js:[16](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:17)0:5)
      at async ScriptTransformer.requireAndTranspileModule (/home/runner/work/directus/directus/node_modules/.pnpm/@jest+transform@29.5.0/node_modules/@jest/transform/build/ScriptTransformer.js:805:16)
      at async runGlobalHook (/home/runner/work/directus/directus/node_modules/.pnpm/@jest+core@29.5.0/node_modules/@jest/core/build/runGlobalHook.js:101:9)
      at async runJest (/home/runner/work/directus/directus/node_modules/.pnpm/@jest+core@29.5.0/node_modules/@jest/core/build/runJest.js:327:5)
      at async _run10000 (/home/runner/work/directus/directus/node_modules/.pnpm/@jest+core@29.5.0/node_modules/@jest/core/build/cli/index.js:344:7)
      at async runCLI (/home/runner/work/directus/directus/node_modules/.pnpm/@jest+core@29.5.0/node_modules/@jest/core/build/cli/index.js:[19](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:20)9:3)
  [25](https://github.com/directus/directus/actions/runs/5076878447/jobs/9119493233#step:7:26)hundefined
  /home/runner/work/directus/directus/tests/blackbox:
   ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  tests-blackbox@ test: `jest`
  Exit status 1
   ELIFECYCLE  Command failed with exit code 1.
  Error: Process completed with exit code 1.
  ```
</details>
